### PR TITLE
Awesome books: Navigation

### PR DESCRIPTION
**Pull request by SamTush**
### In this pull request, these are the changes made:

- Changing the display of three sections using javascript
- Add books to list section
- Remove a book from list section
- Add contact section

>[Live demo](https://samtush.github.io/Awesome-books/)

### CODE REVIEW

File: dist/css/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    background-color: #012622;
+  }
+  header .date-time .time {
+    display: flex;
+    justify-content: start;
```
Comment (binhussen) [inline dist/css/style.css:100]: - Great Job for styling it with beautiful UI and making it responseive👍👍 however your input fields are covering the hole and add button is aligned on the left Please update it to have a better design👌👌
Screenshot 2023-02-23 at 12 12 15 PM
- Please also arrange the lists of books like stable as the given wireframe 👍
Screenshot 2023-02-23 at 12 12 44 PM
given wireframe
Screenshot 2023-02-23 at 12 11 37 PM

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @SamTush and @ab-noori , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/css/style.css (hunk 1 above)
Comment (ab-noori) [reply dist/css/style.css:100]: Thanks, Mr. @binhussen for your constructive comments.
We will consider and apply them.

**Review by Munsa1 — APPROVED** [review]
Hi @SamTush @ab-noori , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**